src/BluetoothHciSocket.h

```diff
@@ -39,7 +39,7 @@ class BluetoothHciSocket : public node::ObjectWrap {
 
   void poll();
 
-  void emitErrnoError();
+  void emitErrnoError(const char *syscall);
```

Comment:
may a default parms used here to preserve API, like null ?

Reply:
Doesn't matter really imo - there's two calls to this and it's a private function anyway.